### PR TITLE
docs(napi): Remove JSON.parse from example

### PR DESCRIPTION
**Pull request by ottomated**

### CODE REVIEW

**Comment by ottomated** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6836?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#6836** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6836?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#6834** <a href="https://app.graphite.dev/github/pr/oxc-project/oxc/6834?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @ottomated and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by overlookmotel — APPROVED** [review]
Great. But we should not merge this until #6796 is merged.

Marking this PR as draft so it doesn't get merged accidentally.

**Pull request merged**
